autopilot/sagemaker_autopilot_abalone_parquet_input.ipynb

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
<p>This notebook exhibits the use of a Parquet dataset -&gt; This is an example of how to use dataset in Parquet format.</p><p>Not sure about <code>exhibits</code></p><p> </p><p>"This notebook tackles the exact same problem with the same solution, but has been modified for a Parquet input to be trained in SageMaker Autopilot" -&gt; should you maybe extend the other notebook to run on dataset of different format? Maybe drop this line altogether, as over time it might not be accurate.</p><p> </p><p>You could maybe title this notebook as "Data formats supported by SM Autopilot", and mention a few lines about what formats are supported, and a link to documentation. Perhaps this notebook can be later extended once more formats become available. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3093/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Talked offline, we will keep this notebook as is.</p>